src/main/java/baseball/view/InputView.java

```diff
@@ -0,0 +1,109 @@
+package baseball.view;
+
+import baseball.component.Numbers;
+import baseball.exception.message.ExceptionMessage;
+import baseball.exception.noncritical.IllegalInputException;
+
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class InputView {
+    private static final String INPUT_MESSAGE = "숫자를 입력해주세요 : ";
+    private static final String WHITESPACE_REGEX = "\\s+";
+    private static final int CORRECT_NUMBERS_LENGTH = 3;
+    private static final int CORRECT_STRINGS_LENGTH = 1;
+    private final Scanner scanner = new Scanner(System.in);
```

Comment:
static final 로 선언한 상수와 필드로 사용될 변수는 한 칸 띄워주면 보다 읽기 수월할 것 같아요!

```suggestion
    private static final String INPUT_MESSAGE = "숫자를 입력해주세요 : ";
    private static final String WHITESPACE_REGEX = "\\s+";
    private static final int CORRECT_NUMBERS_LENGTH = 3;
    private static final int CORRECT_STRINGS_LENGTH = 1;
    
    private final Scanner scanner = new Scanner(System.in);
```